### PR TITLE
Origin/signup add info frontend

### PR DESCRIPTION
**Pull request by jhubbard2017**

### CODE REVIEW

File: src/app/routes/auth/components/WelcomeContent.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```
Comment (lmartin73) [inline src/app/routes/auth/components/WelcomeContent.js]: remove this page

File: src/app/routes/auth/components/DisplayContent.js [hunk 2]
```diff
@@ -18,6 +18,6 @@ export default class DisplayContent extends React.Component {
                     </div>
                 </div>
             </div>
-        )
-    }
+        </div>
```
Comment (lmartin73) [inline src/app/routes/auth/components/DisplayContent.js]: don't need this extra div

File: src/app/routes/profile/containers/CreateProfile.js [hunk 3]
```diff
@@ -0,0 +1,485 @@
...
+        this.photoHandler = this.photoHandler.bind(this);
+
+        // State used to display user input information on step 3 (Verification)
+        // Can not access refs in render method (DOM hasnt been rendered, refs are undefined)
+        this.state = {
+            firstname: "",            // Get first name after registering
+            lastname: "",            // Get last name after registering
+            email: "",     // Get email after registering
+            phone: "",
+            phonetype: "",
```
Comment (lmartin73) [inline src/app/routes/profile/containers/CreateProfile.js]: phone_type

File: src/app/routes/profile/containers/CreateProfile.js [hunk 4]
```diff
@@ -0,0 +1,485 @@
...
+            street1: "",
+            street2: "",
+            city: "",
+            state: "",
+            zip: "",
+            country: "",
+            addresstype: ""
```
Comment (lmartin73) [inline src/app/routes/profile/containers/CreateProfile.js]: address_type

**Review by lmartin73 — CHANGES_REQUESTED** [review]

File: src/app/routes/profile/containers/CreateProfile.js (hunk 4 above)
Comment (jhubbard2017) [reply src/app/routes/profile/containers/CreateProfile.js]: Done

File: src/app/routes/profile/containers/CreateProfile.js (hunk 3 above)
Comment (jhubbard2017) [reply src/app/routes/profile/containers/CreateProfile.js]: Done

File: src/app/routes/auth/components/DisplayContent.js (hunk 2 above)
Comment (jhubbard2017) [reply src/app/routes/auth/components/DisplayContent.js]: Done

File: src/app/routes/auth/components/WelcomeContent.js (hunk 1 above)
Comment (jhubbard2017) [reply src/app/routes/auth/components/WelcomeContent.js]: Done

**Review by lmartin73 — APPROVED** [review]

**Pull request merged**